{
  "uid" : "2600c81fbd393f90",
  "name" : "test_create_posts_process",
  "fullName" : "docs.testcases.test_all_in_one.TestConduit#test_create_posts_process",
  "historyId" : "fd158587032c2f044528f4e02440bac1",
  "time" : {
    "start" : 1628142043447,
    "stop" : 1628142048752,
    "duration" : 5305
  },
  "status" : "broken",
  "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: 'contents.csv'",
  "statusTrace" : "self = <test_all_in_one.TestConduit object at 0x7f425d11bf10>\n\n    def test_create_posts_process(self):\n        basic_login(self.driver)\n    \n        for i in range(1):\n>           with open('contents.csv') as article_file:\nE           FileNotFoundError: [Errno 2] No such file or directory: 'contents.csv'\n\ndocs/testcases/test_all_in_one.py:247: FileNotFoundError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: 'contents.csv'",
    "statusTrace" : "self = <test_all_in_one.TestConduit object at 0x7f425d11bf10>\n\n    def test_create_posts_process(self):\n        basic_login(self.driver)\n    \n        for i in range(1):\n>           with open('contents.csv') as article_file:\nE           FileNotFoundError: [Errno 2] No such file or directory: 'contents.csv'\n\ndocs/testcases/test_all_in_one.py:247: FileNotFoundError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "bdf375812aafe4e2",
      "name" : "log",
      "source" : "bdf375812aafe4e2.txt",
      "type" : "text/plain",
      "size" : 344
    }, {
      "uid" : "4637d7fea96e885a",
      "name" : "stderr",
      "source" : "4637d7fea96e885a.txt",
      "type" : "text/plain",
      "size" : 215
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "docs.testcases"
  }, {
    "name" : "suite",
    "value" : "test_all_in_one"
  }, {
    "name" : "subSuite",
    "value" : "TestConduit"
  }, {
    "name" : "host",
    "value" : "fv-az28-873"
  }, {
    "name" : "thread",
    "value" : "3872-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "docs.testcases.test_all_in_one"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "2600c81fbd393f90.json",
  "parameterValues" : [ ]
}